modules/remove.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable class-methods-use-this */
+import { Storage } from './storage.js';
+
+const storage = new Storage();
+
+export class Remove {
+  constructor() {
+    return null;
+  }
+
+  removeBook(id) {
+    if (id != null) {
+      const oldBooks = storage.fetchBooks();
+
+      const newBooks = oldBooks.filter((book, index) => {
+        if (index !== Number(id)) return true;
+        return null;
+      });
+
+      storage.storeBooks(newBooks);
+      /* eslint-disable no-implied-eval */
+      setTimeout('location.reload(true);', 0);
+      /* eslint-enable no-implied-eval */
+    }
+  }
```

Comment:
- [ ] [OPTIONAL] it would be better if you use es6 `arrow function` instead of `normal method` in order to practice working with both